app/bundles/CampaignBundle/Model/EventModel.php

```diff
@@ -727,7 +727,7 @@ public function executeEvent(
             return false;
         }
 
-        if ($event['eventType'] == 'condition') {
+        if ($event['eventType'] == 'condition' || $event['eventType'] == 'action') {
```

Comment:
I fear this will change behavior as the $allowNegative has a slightly different purpose than just in the context of decisions. This only applies to the scenario that the decision has just non-action events attached or if all of the associated action events completely failed to execute. But in either case, we don't want to transverse the non-action path. So I think the safer way to handle this is to add a check to triggerEvent for the scenario. 

Before https://github.com/mautic/mautic/blob/fbfb4387908a107d7fc1cb286aca3ff9c894a738/app/bundles/CampaignBundle/Model/EventModel.php#L321, insert something like
```
                        } elseif ($child['decisionPath'] == 'no') {
                            // non-action paths should not be processed by this because the contact already took action in order to get here
                            $childrenTriggered = true;
                        }
```